dispatcher.go

```diff
@@ -79,13 +82,18 @@ func NewDispatcher(cfg Config) Dispatcher {
 		panic("a service name is required")
 	}
 
+	if cfg.Tracer == nil {
+		cfg.Tracer = opentracing.NoopTracer{}
+	}
```

Comment:
you could use late binding (as we did in tchannel) if you defer getting the tracer until needed and fall back onto `opentracing.GlobalTracer()` function, rather than instantiating NoopTracer manually.